### DISCUSSION TITLE
React typescript compilation error when importing Static<typeof userSchema> as type

### DISCUSSION THREAD

**Discussion by pipwarrior** [feathersjs/feathers#3175] [Q&A]
### Steps to reproduce
1. On my React app, i imported all interfaces from the FeathersJS webserver into a server-interface.ts file
`export type User = import('../../../../webserver/src/services/users/users.schema').User;`

2. Imported the User interface
`import { User } from '../server-interface';`
`export const createUser = (data: Partial<User>): Promise<User> => (
  userFeathersService.create(data)
);`

React compilation error:
<img width="1111" alt="TypeScript Errors" src="https://user-images.githubusercontent.com/19699070/228428319-5d0261b6-f7d1-47f3-821e-3d68c1f82b31.png">

### Expected behavior

Tell us what should happen

### Actual behavior

Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:

**Operating System**:

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by daffl** [comment]
It looks like your bundler is pulling in the actual file. This shouldn't happen when [importing the typed client itself](https://feathersjs.com/guides/cli/client.html) which also re-exports your application types so you can do:

```ts
import type { UserData, User } from 'my-app'
```
